design/cstor/v1/cstorvolume_policy.md

```diff
@@ -0,0 +1,489 @@
+# Proposed v1 CStorVolumeConfig, CStorVolumePolicy and CStorVolumeAttachment Schema
+
+## Table of Contents
+
+* [Introduction](#introduction)
+* [Goals](#goals)
+* [VolumeConfig Schema Proposal](#volume-config-schema-proposal)
+* [VolumePolicy Schema Proposal](#volume-policy-schema-proposal)
+* [VolumeAttachment Schema Proposal](#volume-attachment-schema-proposal)
+
+## Introduction
+
+This proposal highlights enhancements and migration of current `CStorVolumeConfig` and `CStorVolumePolicy` schema and
```

Comment:
Same as the previous doc. The only goal is to propose the schema of various CRs.